extensions/ql-vscode/src/local-databases-ui.ts

```diff
@@ -379,14 +379,12 @@ export class DatabaseUI extends DisposableObject {
         );
 
         let databaseItem = this.databaseManager.findDatabaseItem(uri);
-        const isTutorialDatabase = true;
         if (databaseItem === undefined) {
           databaseItem = await this.databaseManager.openDatabase(
             progress,
             token,
             uri,
             "CodeQL Tutorial Database",
```

Comment:
I don't particularly like that we're matching on the DisplayName later in the local-databases.ts file, because someone might change it here without thinking about the consequences. Can we add a comment here making this dependency clear?